README.md

```diff
@@ -1,47 +1,18 @@
-# DreamBooth Template
+# Stable Diffusion Dreambooth model template
 
-🦜 Hey! We built an API that automates the whole process outlined below. To get started using that new API, check out the blog post at [replicate.com/blog/dreambooth-api]([https://replicate.com/blog/dreambooth-api])
+This is the template used by [replicate.com dreambooth api](https://replicate.com/blog/dreambooth-api/) to build custom dreambooth models.
 
-If you wish to use the old manual method, you can continue reading below.
+This template is based on [cog-stable-diffusion](https://github.com/replicate/cog-stable-diffusion) which uses diffusers.
 
----
+## How to use this template
 
-This is a template repo for training and publishing your own custom Stable Diffusion model using Replicate.
+Ignore this template and use our [replicate.com dreambooth api](https://replicate.com/blog/dreambooth-api/).  It will generate an HTTP API for you generate images from your model.
 
-It uses [replicate.com/replicate/dreambooth](https://replicate.com/replicate/dreambooth) to train the model using your custom imagery.
+## How to use this template locally
```

Comment:
Maybe this can be consolidated:

```
## Usage

This template is primarily intended for use by [Replicate's DreamBooth API](https://replicate.com/blog/dreambooth-api), which is what you probably want to use to train and publish your own model.

If you really want to use this template locally, you can do so by following these steps:
```